privacyidea/api/validate.py

```diff
@@ -261,9 +270,7 @@ def check():
            Host: example.com
            Accept: application/json
 
-           user=user
-           realm=realm1
-           pass=s3cret123456
+           user=user&realm=realm1&pass=s3cret123456
```

Comment:
Please keep this as new lines and revert. The ampersand is a notation for a GET request (at least this is how we use it in the complete documentation)